app/helpers/active_storage/meta_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveStorage::MetaHelper
+  def active_storage_meta_tags
+    safe_join [
+      tag.meta(name: 'direct-uploads-url', content: rails_direct_uploads_url)
+    ]
```

Comment:
So the meta tag is just there to save repeating the URL on the input tags? That seems like a bit of an over-optimization to me. I'd rather just have it on the input tag alone, such that using this feature doesn't require any setup gymnastics.